### ISSUE TITLE
when uploading scw list file, list of science windows is limited to 50 for registered users

### ISSUE THREAD

**Issue by ferrigno** [oda-hub/dispatcher-app#176]
In general, the rule is 
1. 50 scw for unregistered user
2. 500 scw for registered user
3. unlimited scw for users with HPC unige role

However if one uploads a list of scw, the number is limited to 50 even if you are registered. There should be a unique rule to be applied.

**Comment by ferrigno** [comment]
https://github.com/oda-hub/doc-multi-user/blob/main/plan-roles-users.md ?

**Comment by volodymyrss** [comment]
> https://github.com/oda-hub/doc-multi-user/blob/main/plan-roles-users.md ?

yes?

**Comment by volodymyrss** [comment]
or you mean that actually this link https://github.com/oda-hub/doc-review-guidelines/issues/47

it should be still checked that scwlist from file is correctly checked. with a test

**Comment by ferrigno** [comment]
Yes, I mean that a clarification and discussion is needed on the roles and limits
See
https://github.com/oda-hub/doc-multi-user/issues/4

I wrote the help pages and showed them to you as well this morning with a different assumption on roles and limits.

**Comment by volodymyrss** [comment]
This issue here has something to do with providing the file specifically.  Maybe there is a special case with providing the file. If not, this should be closed. But it should be verified.

**Comment by burnout87** [comment]
I tried to make a request with a file, and the rules on the roles are correctly applied.

**Comment by volodymyrss** [comment]
do you mean that you are able to request up to 500 scw with file list?

**Comment by burnout87** [comment]
Yes, but perhaps I did something wrong. Is there an example file list I can test with?

**Comment by volodymyrss** [comment]
I did not investigate, but could we make a test to make sure that uploading scw with file causes the same effects as providing it otherwise?

**Comment by burnout87** [comment]
Ok, just to for my understanding, is the file expected to have the folloing format?

```
005100410010.001
005100410020.001
005100410030.001
005100410040.001
005100410050.001
005100410060.001
005100410070.001
005100410080.001
005100410090.001
005100410100.001
005100410110.001
005100410120.001
005100410130.001
...
```

**Comment by volodymyrss** [comment]
Maybe, could you verify?

**Comment by burnout87** [comment]
Yes, I just tried and it works. I will write the test(s)

**Comment by volodymyrss** [comment]
just to note, this issue has 3 distinct aspects:

* it is possible that passing scw list with file is a special case. this is the primary remaining purpose of this issue here
* @ferrigno got confused by the role meaning previously discussed and [documented](https://github.com/oda-hub/doc-multi-user/blob/main/plan-roles-users.md)
* roles of @ferrigno  seem not to be what is shown and expected, I made an issue for this https://github.com/oda-hub/dispatcher-app/issues/182

only the first point remains here and will be resolved by the test
